app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -61,17 +87,23 @@ public function __construct(MauticFactory $factory, $modelName, $viewPermissionB
      * @param string              $valueColumn The column that houses the value
      * @param CompositeExpression $expr        Use $factory->getDatabase()->getExpressionBuilder()->andX()
      *
+     * @throws \BadMethodCallException
+     *
      * @return array|void
      */
     public function getTokens(
-        $tokenRegex,
-        $filter = '',
-        $labelColumn = 'name',
-        $valueColumn = 'id',
+        string $tokenRegex,
+        string $filter = '',
+        string $labelColumn = 'name',
+        string $valueColumn = 'id',
```

Comment:
Adding types to existing methods will change their signature so everyone who overwrites the class will get errors. This is a BC change and should be documented in the GDoc. Similarly in the __construct above.